site/content/help/quickstart/index.md

```diff
@@ -28,44 +41,44 @@ description: A guide designed to get you started with yuzu quickly.
 
 ## Hardware Requirements
 
-#### CPU: 
+#### CPU:
 
 Any x86_64 CPU with support for the FMA instruction set. 6 threads or more are recommended.
 
-- Minimum: Intel Core i5-4430 / AMD Ryzen 3 1200
+- **Minimum:** Intel Core i5-4430 / AMD Ryzen 3 1200
 
-- Recommended: Intel Core i5-10400 / AMD Ryzen 5 3600
+- **Recommended:** Intel Core i5-10400 / AMD Ryzen 5 3600
 
-#### Dedicated graphics: 
+#### Dedicated graphics:
 
 OpenGL 4.6 or Vulkan 1.1 compatible hardware and drivers are mandatory. Half-float support and 4GB of VRAM are recommended.
 
-- Minimum for Linux: NVIDIA GeForce GT 1030 2GB / AMD Radeon R7 240 2GB
+- **Minimum for Linux:** NVIDIA GeForce GT 1030 2GB / AMD Radeon R7 240 2GB
 
-- Minimum for Windows: NVIDIA GeForce GT 1030 2GB / AMD Radeon RX 550 2GB
+- **Minimum for Windows:** NVIDIA GeForce GT 1030 2GB / AMD Radeon RX 550 2GB
 
-- Recommended: NVIDIA GeForce GTX 1650 4GB / AMD Radeon RX Vega 56 8GB
+- **Recommended:** NVIDIA GeForce GTX 1650 4GB / AMD Radeon RX Vega 56 8GB
 
 #### Integrated graphics:
 
 Integrated graphics will produce very low performance. A dedicated GPU will produce better results on all scenarios.
 This is only for listing iGPU support.
 
-- Minimum for Linux: Intel HD 5300 / AMD Radeon R5 Graphics
+- **Minimum for Linux:** Intel HD 5300 / AMD Radeon R5 Graphics
 
-- Minimum for Windows: Intel HD Graphics 520 / AMD Radeon Vega 3
+- **Minimum for Windows:** Intel HD Graphics 520 / AMD Radeon Vega 3
 
-- Recommended: Intel UHD Graphics 750 / AMD Radeon Vega 7
+- **Recommended:** Intel UHD Graphics 750 / AMD Radeon Vega 7
 
-#### RAM: 
+#### RAM:
 
 Since an integrated GPU uses system RAM as its video memory (VRAM), our memory requirement in this configuration is higher.
```

Comment:
```suggestion
Since an integrated GPU uses system RAM as its video memory (VRAM), our memory requirement in this configuration is higher. Highest possible frequency is strongly recommended.
```